StripeConnect/StripeConnectTests/Internal/Webview/ConnectWebViewTests.swift

```diff
@@ -112,10 +120,11 @@ class ConnectWebViewTests: XCTestCase {
                                       windowFeatures: .init())
 
         XCTAssertNil(webView)
-        wait(for: [canOpenURLExpectation, openURLExpectation], timeout: 0.1)
+        wait(for: [canOpenURLExpectation, openURLExpectation], timeout: TestHelpers.defaultTimeout)
```

Comment:
I assume this was always intended to use the default timeout? If not, I can change it back.